Modules/_uuidmodule.c

```diff
@@ -0,0 +1,40 @@
+#define PY_SSIZE_T_CLEAN
+
+#include "Python.h"
+#include <uuid/uuid.h>
+
+
+static PyObject *
+_uuid_generate_time_safe(void)
+{
+    uuid_t out;
+    int res;
+
+    res = uuid_generate_time_safe(out);
+    return Py_BuildValue("y#i", (const char *) out, sizeof(out), res);
+}
+
+
+static PyMethodDef uuid_methods[] = {
+    {"generate_time_safe", (PyCFunction)_uuid_generate_time_safe, METH_NOARGS, NULL},
+    {NULL, NULL, 0, NULL}           /* sentinel */
+};
+
+static struct PyModuleDef uuidmodule = {
+    PyModuleDef_HEAD_INIT,
+    "_uuid",
+    NULL,
+    -1,
+    uuid_methods,
```

Comment:
Can you use C99 style here to avoid surprising/boring "NULL," members?

```
.m_name = "_uuid",
.m_size = -1,
.m_methods = uuid_methods,
```

Reply:
Oh, great, I'll do that.